src/components/SideMenu/MenuItem.tsx

```diff
@@ -40,7 +40,12 @@ export class MenuItem extends React.Component<MenuItemProps> {
   render() {
     const { item, withoutChildren } = this.props;
     return (
-      <MenuItemLi onClick={this.activate} depth={item.depth} data-item-id={item.id}>
+      <MenuItemLi
+        onClick={this.activate}
+        depth={item.depth}
+        data-item-id={item.id}
+        topMargin={item.topMargin}
```

Comment:
Remove `topMargin` from this PR at all this, please.

This has nothing to do with this PR.

Also, if you like groups with margin it doesn't mean everyone else does.

Reply:
That is just to separate sections at the end from tag groups. It could be confusing if not:

![image](https://user-images.githubusercontent.com/22495416/86029055-f634ad80-ba32-11ea-9e31-0ecf508adc8c.png)

![image](https://user-images.githubusercontent.com/22495416/86028854-b8378980-ba32-11ea-8e6f-4b116cd809f0.png)

Reply:
Ahh, I see. Let me think on alternative idea of fixing it. I will get back to this tomorrow.

Reply:
Hi! Any new idea on this change? Thank you.

Reply:
Sorry fo the delay in response. 

I tried with a separator and I don't like the result:

![image](https://user-images.githubusercontent.com/22495416/109685483-96724180-7b81-11eb-8df9-78aa150c462c.png)

What I did is the following the same flow than for topMargin, but building a modified linear-gradient. Actually, no too smart:
```
background-image: linear-gradient(to right, transparent 0px, transparent 20px, 20px, #e1e1e1 240px, 240px, transparent 260px);
background-size: 100% 1px;
background-repeat: no-repeat;
background-position: top;
```

I do not like this solution, I still prefer an space between sections and categories.

What do you think that could be done here?